config/examples/Elegoo/Neptune 2/Configuration.h

```diff
@@ -680,15 +646,15 @@
 // Above this temperature the heater will be switched off.
 // This can protect components from overheating, but NOT from shorts and failures.
 // (Use MINTEMP for thermistor short/failure protection.)
-#define HEATER_0_MAXTEMP 270
+#define HEATER_0_MAXTEMP 275
```

Comment:
If the heater is lined with PTFE tubing, such high temperatures may release some bad smoke. Even 270 is pretty high. Does this machine have an all-metal hotend?